src/inc/corinfo.h

```diff
@@ -3074,4 +3074,19 @@ class ICorDynamicInfo : public ICorStaticInfo
 #define IMAGE_REL_BASED_REL32           0x10
 #define IMAGE_REL_BASED_THUMB_BRANCH24  0x13
 
+// The identifier for ARM32-specific PC-relative address
+// computation corresponds to the following instruction
+// sequence:
+//  l0: movw rX, #imm_lo  // 4 byte
+//  l4: movt rX, #imm_hi  // 4 byte
+//  l8: add  rX, pc <- after this instruction rX = relocTarget
+//
+// Program counter at l8 is address of l8 + 4
+// Address of relocated movw/movt is l0
+// So, imm should be calculated as the following:
+//  imm = relocTarget - (l8 + 4) = relocTarget - (l0 + 8 + 4) = (relocTarget - l_0 - 12)
+//
+#define IMAGE_REL_BASED_REL_THUMB_MOV32_PCREL   0x14
+#define IMAGE_REL_BASED_REL_THUMB_MOV32_PCREL_CORRECTION (12)
```

Comment:
Fixed. I removed the preprocessor definition, and replaced it with function's local constant:
```
            // For details about how the value is calculated, see
            // description of IMAGE_REL_BASED_REL_THUMB_MOV32_PCREL
            const UINT32 offsetCorrection = 12;

            UINT32 imm32 = pActualTarget - (pSite + offsetCorrection);

            PutThumb2Mov32((UINT16 *)pLocation, imm32);

```